lua/grpc.lua

```diff
@@ -200,13 +203,29 @@ end
 local MISSION_ENV = 1
 local HOOK_ENV = 2
 
+-- determine interval based on throughput limit
+local interval = 0.02
+if GRPC.throughputLimit <= 2 then
+  interval = 1.0
+elseif GRPC.throughputLimit <= 60 then
+  interval = 0.25
+elseif GRPC.throughputLimit <= 120 then
+  interval = 0.125
+elseif GRPC.throughputLimit <= 240 then
+  interval = 0.06
+elseif GRPC.throughputLimit <= 480 then
+  interval = 0.03
+end
```

Comment:
Is there a reason you are doing boundaries instead of a matematical calculation? I am not fussed either way, just not sure the pros and cons of each approach you might know of.

Reply:
A mathematical calculation sounds better to me, but I am rather bad at coming up with one

Reply:
Changed it to a formula in 

It is roughly the same as before. The `min`/`max` are there to clamp the value in a range between `0.03` and `1.0`s.

![image](https://user-images.githubusercontent.com/409021/137201030-85e8ca29-dcf5-42fc-b805-592643af8b65.png)